source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/ImageRendererTests.swift

```diff
@@ -78,6 +78,23 @@ class ImageRendererTests: XCTestCase {
         XCTAssertEqual(contentView.imageProperties?.contentSize.width, CGFloat(truncating: width))
     }
     
+    func testRendererSetsPixelHeight() {
+        let height : NSNumber = 50
+        fakeImageView = .make(pixelHeight: height)
+        
+        let contentView = renderImageView()
+        XCTAssertEqual(contentView.imageProperties?.contentSize.height, CGFloat(truncating: height))
+    }
+    
+    func testRendererSetsExplicitDimensionsWhenImageSizeIsAlsoGiven() {
+        let width : NSNumber = 50
+        fakeImageView = .make(imageSize: .large, pixelWidth: width)
+        
+        let contentView = renderImageView()
+        XCTAssertEqual(contentView.imageProperties?.contentSize.width, CGFloat(truncating: width))
+    }
+    
+    
```

Comment:
This is great! Maybe we should add tests like these to catch regressions in this area.

Reply:
Sure. I will update some more.